doc/api/stream.md

```diff
@@ -505,6 +505,16 @@ Is `true` after [`writable.end()`][] has been called. This property
 does not indicate whether the data has been flushed, for this use
 [`writable.writableFinished`][] instead.
 
+##### writable.writableCorked
+<!-- YAML
+added: REPLACEME
+-->
+
+* {Integer}
```

Comment:
`tools/doc/type-parser.js` expects `integer` rather than `Integer` so this is causing the docs to fail to build.

```suggestion
* {integer}
```